test/parallel/test-trace-events-environment.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+if (!common.isMainThread)
+  common.skip('process.chdir is not available in Workers');
```

Comment:
Does that also apply to `cwd` for `child_process.fork` (as an alternative)?

Reply:
It shouldn't, but that's a question for @addaleax and is a separate concern from this PR.

Reply:
When using `cwd`, `chdir()` is executed in the child process, so, yes, I think this test could be written without `process.chdir()` and the need to skip it in Workers.